docs/samples/fluid/README.md

```diff
@@ -0,0 +1,337 @@
+
+curl -v -H "Content-Type: application/json" -H "Host: ${SERVICE_HOSTNAME}" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/bloom:predict" -d '{"prompt": "It was a dark and stormy night", "result_length": 50}'
+```
+
+Output:
+
+```
+❯ curl -v -H "Content-Type: application/json" -H "Host: ${SERVICE_HOSTNAME}" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/bloom:predict" -d '{"prompt": "It was a dark and stormy night", "result_length": 50}'
+*   Trying 52.29.43.253:80...
+* Connected to a38c4xxxxxxxxxxxxxxx717177.eu-central-1.elb.amazonaws.com (52.29.43.253) port 80 (#0)
+> POST /v1/models/bloom:predict HTTP/1.1
+> Host: fluid-bloom.kserve-fluid-demo.a38c4xxxxxxxxxxxxxxx717177.eu-central-1.elb.amazonaws.com
+> User-Agent: curl/7.86.0
+> Accept: */*
+> Content-Type: application/json
+> Content-Length: 65
+>
+* Mark bundle as not supporting multiuse
+< HTTP/1.1 200 OK
+< content-length: 190
+< content-type: application/json
+< date: Wed, 12 Apr 2023 15:46:16 GMT
+< server: istio-envoy
+< x-envoy-upstream-service-time: 52071
+<
+{
+  "result": "It was a dark and stormy night, and the wind blew with a\nfearful howl. The moon was obscured by a thick cloud, and the stars\nshone dimly. The wind blew the snow from the"
+}
+* Connection #0 to host a38c4xxxxxxxxxxxxxxx717177.eu-central-1.elb.amazonaws.com left intact
+```
+
+### Performance Benchmark
+
+To conduct a performance benchmark, we will compare the scaling time of the inference service with Storage initializer and Fluid across different model artifacts, measuring the time it takes for the service to scale from `0 to 1`.
+
+We will use the following [machine types](https://aws.amazon.com/ec2/instance-types/m5/) for our tests:
+
+* Data: m5.xlarge
+* Workload: m5.2xlarge, m5.4xlarge
+
+We will use the following Fluid runtimes for our tests:
+
+* JindoFS
+
+For the purposes of our benchmark, we will assume that nodes are readily available and that images are cached in the nodes, so we will not be factoring in node provisioning time or image pulling time.
+
+Other assumptions:
+
+* S3 bucket and Cluster are in the same region (eu-central-1)
+* Data are preloaded in workers
+
+
+Command to measure the total time:
+
+```sh
+# Total time includes: pod initialization and running + download model (storage initializer) + load model + inference + network
+curl --connect-timeout 3600 --max-time 3600 -o /dev/null -s -w 'Total: %{time_total}s\n' -H "Content-Type: application/json" -H "Host: ${SERVICE_HOSTNAME}" "http://${INGRESS_HOST}:${INGRESS_PORT}/v1/models/bloom:predict" -d '{"prompt": "It was a dark and stormy night", "result_length": 50}'
+# Total time: 53.037210s, status code: 200
+```
+
+| Model Name                                                                | Model Size (Snapshot) | Machine Type | KServe + Storage Initializer                         | KServe + Fluid(JindoFS)                    |
```

Comment:
@lizzzcai did you try out juicefs which seems like  a good choice as well?
https://juicefs.com/docs/community/comparison/juicefs_vs_alluxio

Reply:
Hi @yuzisun I didn't try it out here. One thing need to know for [juicefs](https://juicefs.com/docs/community/architecture) is you have to provide Data Storage (like S3) and Metadata Engine (like redis) as dependencies to set it up. I will give it a try as well, but like alluxio, maybe we can do it later. (I will add a link to my own [repo](https://github.com/lizzzcai/kserve-fluid-demo#performance-benchmark), and update it to this example once the data are reviewed)